multiqc/modules/kat/kat.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python
+
+""" MultiQC module to parse output from KAT """
+import logging
+from collections import OrderedDict
+import json
+
+from multiqc.modules.base_module import BaseMultiqcModule
+from multiqc.plots import table
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+
+class MultiqcModule(BaseMultiqcModule):
+    def __init__(self):
+        # Initialise the parent object
+        super(MultiqcModule, self).__init__(name='KAT', anchor='kat',
+                                            href="https://github.com/TGAC/KAT",
+                                            info="is an toolkit for analysing sequencing data via its k-mer spectra.")
+
+        # Find and load any KAT dist analysis reports
+        self.kat_data = dict()
+        for c_file in self.find_log_files('kat'):
+            s_name = self.clean_s_name(c_file['s_name'][:-14], c_file['root'])
+            content = json.loads(c_file['f'])
+            self.kat_data[s_name] = self.parse_kat_report(content)
+
+        # Filter to strip out ignored sample names
+        self.kat_data = self.ignore_samples(self.kat_data)
+
+        if len(self.kat_data) == 0:
+            raise UserWarning
+
+        log.info("Found {} reports".format(len(self.kat_data)))
+
+        # Write parsed report data to a file
+        self.write_data_file(self.kat_data, 'multiqc_kat')
+
+        headers = OrderedDict()
+        headers["kmer_peaks"] = {
+            'title': '# of Kmer Peaks',
+            'description': "Number of peaks identified in the K-mer spectra",
+            'format': '{:,.0f}',
+            'custom_style': 'width=10%'
+        }
+        headers["gc_peaks"] = {
+            'title': '# of GC Peaks',
+            'description': "Number of peaks identified in the GC distribution",
+            'format': '{:,.0f}',
+            'custom_style': 'width=10%'
+        }
+        headers["est_genome_size"] = {
+            'title': 'Est. genome Size',
+            'description': "Estimated Genome Size based on K-mer spectra",
+            'format': '{:,.0f}',
+            'custom_style': 'width=20%'
+        }
+        headers["mean_kmer_freq"] = {
+            'title': 'Mean K-mer Freq.',
+            'description': "Mean K-mer Frequency, provides an estimate of sequencing coverage",
+            'format': '{:,.0f}',
+            'custom_style': 'width=20%',
+            'suffix': 'x'
+        }
+
+        kat_config = {
+            'namespace': 'KAT',
+            'scale': 'RdYlGn',
+            'save_file': True,
```

Comment:
We're already saving the data from the module, so I'm not sure that we need to save the table as well..